nixie/cli.py

```
@@ -113,6 +113,7 @@ async def render_block(
     path: Path,
     idx: int,
     semaphore: asyncio.Semaphore,
+    verbose: bool | None = None,
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)